coloringbook/mail/utilities.py

```diff
@@ -338,13 +340,15 @@ def send_email(recipient, survey_data, survey, immediate=False):
     csv_files = create_survey_results_csv(csv_data)
 
     # Only for testing purposes.
```

Comment:
This comment needs to be inserted before the new line 344. I was confused for a minute before I understood that the call to `send_async_email` is not supposed to run in normal operation.

Reply:
It confused me as well looking at the code just now. 🥲  I've moved it!